tmt/tests/test-00-bootc-install.sh

```diff
@@ -36,6 +36,23 @@ if [ "$TMT_REBOOT_COUNT" -eq 0 ]; then
     ls -al "$BOOTC_TEMPDIR"
 
     pushd "$TMT_TREE"
+        find /etc/yum.repos.d/*.repo -name tag-repository.repo | xargs -I '{}' cp -v '{}' repos.d/
```

Comment:
Do you want to copy /etc/yum.repos.d/tag-repository.repo, or any tag-repository.repo file in tree of /etc/yum.repos.d? I guess the first. What about:

~~~
if test -e /etc/yum.repos.d/tag-repository.repo; then
  cp -v /etc/yum.repos.d/tag-repository.repo repos.d
fi
~~~

Reply:
You're right, this improves readability.